resources/rpm/project.spec.erb

```diff
@@ -233,7 +232,9 @@ fi
       chkconfig --del <%= service.name %> || :
     fi
   <%- elsif @platform.servicetype == "aix" -%>
-    /usr/bin/stopsrc -s <%= service.name -%> > /dev/null 2>&1 || :
+    if [ $1 = 0 ]; then
```

Comment:
Because consistency is the last refuge of the unimaginative.

Or, you know, because we're awesome :)